app/kinesis/ProductionMetricsIndexStreamReader.scala

```diff
@@ -40,7 +51,41 @@ object ProductionMetricsStreamReader {
 
     val codec = KinesisEvent
 
-    override protected def processEvent(eventWithSize: EventWithSize[KinesisEvent]): Unit =
-      println(s"Processed event ${eventWithSize.event}")
+    override protected def processEvent(eventWithSize: EventWithSize[KinesisEvent]): Unit = {
+      val event = eventWithSize.event
+      val eventType = event.eventType
+      eventType match {
+        case CapiContent => processCapiEvent(event.eventJson)
+        case _ => Logger.error(s"Invalid event type on kinesis stream could not be processed ${event}")
+      }
+    }
+
+    private def convertStringToDateTime(dateTime: String): Option[DateTime] = {
```

Comment:
Would it be useful to pull this out in a helper file? It has potential to be reused, especially the DateTimeFormat that we're using in `Metrics.scala` as well.